recipes/flask-principal/14346d1.patch

```diff
@@ -0,0 +1,7 @@
+diff --git MANIFEST.in MANIFEST.in
+new file mode 100644
+index 0000000..1aba38f
+--- /dev/null
++++ b/MANIFEST.in
+@@ -0,0 +1 @@
++include LICENSE
```

Comment:
If only... 😩 They need this file to generate the `sdist` though.

Let's go ahead drop this. Sorry for the headache.